config/routes.rb

```diff
@@ -1,6 +1,24 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'forums#index'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+   # User session routes
+  post '/users/:id/logon', to: 'users#logon', as: 'user_logon'
+  delete '/users/logoff', to: 'users#logoff', as: 'user_logoff'
+
+  # Users routes
+  get '/users', to: 'users#index', as: 'users'
+  get '/users/new', to: 'users#new', as: 'new_user'
+  get '/users/:id', to: 'users#show', as: 'user'
+  get '/users/:id/edit', to: 'users#edit', as: 'edit_user'
+  post '/users', to: 'users#create'
+  patch '/users/:id', to: 'users#update'
+  delete '/users/:id', to: 'users#delete'
```

Comment:
change line 15 to   delete '/users/:id', to: 'users#destroy' it can't find the delete action in the controller because its named destroy